ssmconfig_test.go

```diff
@@ -40,6 +46,7 @@ func TestProvider_Process(t *testing.T) {
 			F641    float64 `ssm:"/float64/f641"`
 			F642    float64 `ssm:"/float64/f642" default:"42.42"`
 			Invalid string
+			D       d
```

Comment:
Please also add a pointer to `D` to ensure pointers to structs also populate as expected.

```go
var s struct {
    ...
   D1 d
   D2 *d
}
```